src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -905,7 +909,9 @@ private function registerTemplatingConfiguration(array $config, ContainerBuilder
     {
         $loader->load('templating.xml');
 
-        $container->setParameter('fragment.renderer.hinclude.global_template', $config['hinclude_default_template']);
+        if (!$container->hasParameter('fragment.renderer.hinclude.global_template')) {
```

Comment:
If will never be set as `registerTemplatingConfiguration` is called before `registerFragmentsConfiguration`